core/src/banking_stage.rs

```diff
@@ -705,9 +709,24 @@ mod external {
             );
             assert!(workers.len() <= BankingStage::max_num_workers().get());
 
-            // Spawn vote worker.
+            // Potentially spawn vote worker.
             let mut threads = Vec::with_capacity(workers.len() + 3);
-            threads.push(self.spawn_vote_worker());
+            let tpu_to_pack_receivers = match flags & logon_flags::REROUTE_VOTES {
```

Comment:
Seems much clearer to just use an if flags & logon_flags::REROUTE_VOTES != 0?

Reply:
Updated to use if/else without ranges